spec/models/subscriber_list_spec.rb

```diff
@@ -120,4 +120,78 @@
       )
     end
   end
+
+  describe "#to_json for a record without a type" do
+    subject { SubscriberList.new }
+
+    it "returns a hash as a subscriber list" do
+      assert_nil subject.type
+
+      assert subject.to_json.is_a?(String)
+      parsed_json = JSON.parse(subject.to_json)
+      assert parsed_json.has_key?("subscriber_list")
+      assert_equal %w(id
+                      title
+                      created_at
+                      updated_at
+                      document_type
+                      tags
+                      links
+                      email_document_supertype
+                      government_document_supertype
+                      slug
+                      subscription_url
+                      gov_delivery_id
+                      active_subscriptions_count),
+                   parsed_json["subscriber_list"].keys
```

Comment:
could we extract these out into a single method, as they're the same each time?

```ruby
assert_subscriber_list(parsed_json)

def assert_subscriber_list(parsed_json)
      assert_equal %w(id
                      title
                      created_at
                      updated_at
                      document_type
                      tags
                      links
                      email_document_supertype
                      government_document_supertype
                      slug
                      subscription_url
                      gov_delivery_id
                      active_subscriptions_count),
                   parsed_json["subscriber_list"].keys
end
```

Reply:
👍

Reply:
I think refactoring out the set-up is ok, but I think that having a shared expectation makes the test harder to read. 
Perhaps just refactor the long list of keys in to shared variable?